crates/topos/src/config/tce.rs

```diff
@@ -52,28 +52,28 @@ fn default_db_path() -> PathBuf {
 
 const fn default_libp2p_api_addr() -> SocketAddr {
     SocketAddr::V4(std::net::SocketAddrV4::new(
-        std::net::Ipv4Addr::new(0, 0, 0, 1),
+        std::net::Ipv4Addr::new(0, 0, 0, 0),
```

Comment:
Can't these be `const` these days?

Reply:
Yes it does. I also think that we need to merge this PR as soon as possible to have this kind of update in a unitary fashion